stdlib/random.pyi

```diff
@@ -154,9 +159,16 @@ def triangular(low: float = ..., high: float = ..., mode: float | None = ...) ->
 def betavariate(alpha: float, beta: float) -> float: ...
 def expovariate(lambd: float) -> float: ...
 def gammavariate(alpha: float, beta: float) -> float: ...
-def gauss(mu: float, sigma: float) -> float: ...
+
+if sys.version_info >= (3, 11):
+    def gauss(mu: float = ..., sigma: float = ...) -> float: ...
+    def normalvariate(mu: float = ..., sigma: float = ...) -> float: ...
+
+else:
+    def gauss(mu: float, sigma: float) -> float: ...
+    def normalvariate(mu: float, sigma: float) -> float: ...
```

Comment:
I wonder if replacing these with `pkg_resources.WorkingSet` style aliases would be possible, to better match to runtime.

Reply:
Good point, we should do that. I think for a long time pytype did not support bound methods, but now it does.

Reply:
Yup, seems like we should be able to do this now: https://github.com/python/typeshed/pull/4817#issuecomment-744091994